lib/core-classes/DeviceService.ts

```diff
@@ -373,6 +375,35 @@ export class DeviceService {
     return results;
   }
 
+  async importDevices(devices: JSONObject,
+    { strict, options }: { strict?: boolean, options?: JSONObject }) {
+      const results = {
```

Comment:
Usually we indent like this when arguments does not fit into a single line
```suggestion
  async importDevices(
    devices: JSONObject,
    { strict, options }: { strict?: boolean, options?: JSONObject },
  ) {
      const results = {
```